src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -407,7 +407,8 @@ private async Task<PackageSpec> GetPackageSpecAsync(ISettings settings)
                     RestoreLockProperties = new RestoreLockProperties(
                         await _vsProjectAdapter.GetRestorePackagesWithLockFileAsync(),
                         await _vsProjectAdapter.GetNuGetLockFilePathAsync(),
-                        await _vsProjectAdapter.IsRestoreLockedAsync())
+                        await _vsProjectAdapter.IsRestoreLockedAsync(),
+                        await _vsProjectAdapter.ShouldLockFileBeIgnoredAsync())
```

Comment:
@rrelyea had some concerns abou whether this should actually be consumed in VS. 

We should discuss this before merging.